tests/functional/test_install_reqs.py

```diff
@@ -249,6 +253,53 @@ def test_wheel_user_with_prefix_in_pydistutils_cfg(
     assert 'installed requiresupper' in result.stdout
 
 
+def test_nowheel_user_with_prefix_in_pydistutils_cfg(script, data, virtualenv):
+    virtualenv.system_site_packages = True
+    with open(os.path.join(get_home(), ".pydistutils.cfg"), "w") as cfg:
+        cfg.write(textwrap.dedent("""
+            [install]
+            prefix=%s""" % script.scratch_path))
+
+    result = script.pip('install', '--no-binary=:all:', '--user', '--no-index',
+                        '-f', data.find_links, 'requiresupper',
+                        expect_stderr=True)
+    assert 'installed requiresupper' in result.stdout
+
+
+@pytest.mark.network
+def test_wheel_target_with_prefix_in_pydistutils_cfg(
+        script, data, virtualenv, common_wheels):
+    # pip needs `wheel` to build `requiresupper` wheel before installing
+    script.pip('install', 'wheel', '--no-index', '-f', common_wheels)
+    with open(os.path.join(get_home(), ".pydistutils.cfg"), "w") as cfg:
+        cfg.write(textwrap.dedent("""
+            [install]
+            prefix=%s""" % script.scratch_path))
+
+    target_path = script.scratch_path / 'target'
+    result = script.pip('install', '--target', target_path, '--no-index',
+                        '-f', data.find_links, '-f', common_wheels,
+                        'requiresupper')
+    # Check that we are really installing a wheel
+    assert 'Running setup.py install for requiresupper' not in result.stdout
+    assert 'installed requiresupper' in result.stdout
+
+
+def test_nowheel_target_with_prefix_in_pydistutils_cfg(script, data,
+                                                       virtualenv):
+    with open(os.path.join(get_home(), ".pydistutils.cfg"), "w") as cfg:
```

Comment:
... and again.

Better, why not set up a global name in the file

```python
distutils_cfg = os.path.expanduser('~/.pydistutils.cfg')
```

and use that?

Reply:
Okay.